.env

```diff
@@ -1,6 +1,10 @@
 COMPOSE_PROJECT_NAME=xpub
 XPUB_VERSION=487fe36514623e9171767a637c1f91e505c692d5
+SCIENCEBEAM_TAG=f05c51adbc49b47a65df1c3b7f69d25e401f5560
```

Comment:
The update process for `sciencebeam` will look like:
- open a manual PR to this repo
- review&merge
- propagate this across environments
all while the version of `elife-xpub` remains the same as before (as it is automatically committed at the moment, so it cannot be stopped from propagating). With enough backward compatibility this should not be a problem.

Reply:
Should I add this procedure to the `continuum-playbook` ? ... did we agree where the runbook for xpub will go (see https://github.com/elifesciences/elife-xpub/issues/617#issuecomment-423524784 ) ?

Reply:
Ah yes, I forgot we said we would do this with a tag on the xpub image. That would be better. Can you give us some pointers on how to do that?

Reply:
I think this procedure is good enough in the trade-off between complexity and safety. We would have to make part of `.env` (or some separate file) updated by the formula to make the tag-as-label approach work here.